lib/src/messages/responses/cache/control/list_caches_response.dart

```diff
@@ -35,6 +35,11 @@ class ListCachesSuccess implements ListCachesResponse {
   }
 
   List<String> get cacheNames => caches.map((cache) => cache.name).toList();
+
+  @override
+  String toString() {
+    return "$runtimeType: $cacheNames";
```

Comment:
I'd recommend you take advantage of the base class `toString` throughout these overrides so if we ever decide we want to change the base format in any way we only have to do it in one place:
```suggestion
    return "${super.toString()}: $cacheNames";
```